server/src/test/scala/io/delta/sharing/server/DeltaSharingServiceSuite.scala

```diff
@@ -426,7 +442,7 @@ class DeltaSharingServiceSuite extends FunSuite with BeforeAndAfterAll {
   integrationTest("getTableVersion - get exceptions") {
     // timestamp can be any string here, it's resolved in DeltaSharedTableLoader
     assertHttpError(
-      url = requestPath("/shares/share2/schemas/default/tables/table2?startingTimestamp=abc"),
+      url = requestPath("/shares/share2/schemas/default/tables/table2/version?startingTimestamp=abc"),
```

Comment:
Question: why is version needed?

Reply:
This is to leave the endpoint without `/version` for getTable.  And this is one of the missing tests not updated in that PR.
Now getTable is not needed, I think it's a good idea to have `/version` suffix for getTableVersion.